MAD tank balance changes/overhall

Ethal-A opened the pull request:
the Mad tank has been changed so that it is more useful as a support infantry unit that still causes devastation
video: https://vid.me/CXgI

Commit:
Changed MadTank Weapon for balancing

I significantly changed the mad tanks weapon so that it is more balanced
example: after 8 ticks the mad tank will damage units 1 square away, after 16 the mad tank will damage units 2 squares away, after 24 the mad tank will damage units 3 squares away etc. untill 56 ticks. the damage precentage from thumping is now 2
Video: https://vid.me/CXgI

Commit:
changed mad tank cost & traits

these changes are made so that it fits with the other changes made to the weapon so that its balanced

cjshmyr commented:
I think I'd be willing to accept that with a few tweaks:
- Less crater intervals, so they don't appear so thick by the end in the full circle.
- Not sure about the damage yet, haven't tested it.

Commit:
changed build pallete order TD

changed it so that it follows the tech tree
images of it can be found at: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19220&postdays=0&postorder=asc&vote=viewresult

GraionDilach commented on mods/ra/weapons/other.yaml:
```diff
@@ -263,10 +263,69 @@ Mandible:
 	InvalidTargets: MADTank, Infantry
 	Warhead@1Dam: HealthPercentageDamage
+		Spread: 1c0
+		damage: 2
```
Damage is with a capital D.

GraionDilach commented on mods/ra/weapons/other.yaml:
```diff
@@ -263,10 +263,69 @@ Mandible:
 	Warhead@1Dam: HealthPercentageDamage
+		Spread: 1c0
+		damage: 2
+			Delay: 8
```
The indentation is wrong here - notsure if this'll even work with that.

Phrohdoh replied:
It will not work with incorrect indentation.

Commit:
changed build pallete order TD

changed it so that it follows the tech tree
images of it can be found at: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19220&postdays=0&postorder=asc&vote=viewresult

Truth99A commented:
lets forget this happened ok?
(i have no idea how to use github lmao and in my attempts to remove the commits I messed up a tone of things to the point of no return and may need to completely delete my repository)
as for now go on the other one pull request with the exact same name
